src/react/hooks/useMODUKComponent/useMODUKComponent.ts

```diff
@@ -28,6 +28,11 @@ export function useMODUKComponent<
   useEffect(() => {
     // Initialise the component if and only if the DOM element has changed
     if (ref.current && ref.current !== previousRef.current) {
+      const body = document.getElementsByTagName('body')[0]
+      if (!body) {
+        throw new Error('Could not find body element')
+      }
```

Comment:
Does `document.body` work here?

```suggestion
      const { body } = document
```